scripts/c3_to_s3_rolling.py

```diff
@@ -183,6 +183,13 @@ def get_common_message_attributes(stac_doc: Dict) -> Dict:
             ),
         }
 
+    maturity = dicttoolz.get_in(["properties", "dea:dataset_maturity"], stac_doc)
+    if maturity:
+        msg_attributes["maturity"] = {
+            "DataType": "String",
+            "StringValue": dicttoolz.get_in(["properties", "dea:dataset_maturity"], stac_doc),
+        }
+
```

Comment:
```suggestion
    maturity = dicttoolz.get_in(["properties", "dea:dataset_maturity"], stac_doc)
    if maturity:
        msg_attributes["maturity"] = {
            "DataType": "String",
            "StringValue": maturity
        }

```

No need to get it twice